tests/index.html

```diff
@@ -33,7 +33,9 @@
     <div id="qunit"></div>
     <div id="qunit-fixture">
       <div id="ember-testing-container">
-        <div id="ember-testing"></div>
+        <div id="ember-testing">
+          <div class="ember-view" id="for-basic-dropdown-getDestinationId"></div>
```

Comment:
This is a workaround to an odd issue that was happening with PowerSelect (specifically ember-basic-dropdown) after turning `application-template-wrapper` to false. Some integration tests were failing because [ember-basic-dropdown is looking for a specific wrapper div in tests](https://github.com/cibernox/ember-basic-dropdown/blob/v3.1.0/addon/components/basic-dropdown.ts#L323) that is no longer available after the application-template-wrapper flag is turned off. Big thank you to @adlius for tracking this down!